showcase/gapic-showcase/pom.xml

```diff
@@ -229,5 +229,13 @@
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Even though it is a test jar that is only used in showcase, we may still want to manage it in a centralized place like [here](https://github.com/googleapis/sdk-platform-java/blob/main/gapic-generator-java/pom.xml), WDYT @suztomo?

Reply:
Use the google-cloud-shared-dependencies. Then it matches what a normal client library would work.

Reply:
Thanks - I added the test-jar reference to shared dependencies